gulpfile.js

```diff
@@ -1 +0,0 @@
-eval(require("typescript").transpile(require("fs").readFileSync("./gulpfile.ts").toString()));
```

Comment:
@pleerock how does this work now? Does `gulp-cli` support `gulpfile.ts` files out of the box now?